model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -41,21 +45,50 @@ data class OrtConfiguration(
          * 1. [Command line arguments][args]
          * 2. [Configuration file][configFile]
          * 3. default.conf from resources
+         *
+         * The configuration file is optional and does not have to exist. However, if it exists, but does not
+         * contain a valid configuration, an [IllegalArgumentException] is thrown.
          */
-        fun load(args: Map<String, String> = emptyMap(), configFile: File? = null): OrtConfiguration {
-            val argsConfig = ConfigFactory.parseMap(args, "Command line").withOnlyPath("ort")
-            val fileConfig = configFile?.let {
-                ConfigFactory.parseFile(it).withOnlyPath("ort")
-            }
-            val defaultConfig = ConfigFactory.parseResources("default.conf")
+        fun load(args: Map<String, String> = emptyMap(), configFile: File): OrtConfiguration {
+            val result = ConfigLoader.Builder()
+                .addSource(argumentsSource(args))
+                .addSource(PropertySource.file(configFile, optional = true))
```

Comment:
Please add the "default.conf" as soure with the lowest priority as it was before and as the function docs say. It does currently not contain any values, but this will likely change in the future.

Reply:
Done.